app/views/govuk_component/breadcrumbs.raw.html.erb

```diff
@@ -1,18 +1,20 @@
 <%
   breadcrumbs ||= []
   collapse_on_mobile ||= false
+  inverse ||= false
+  collapse_class =  collapse_on_mobile && breadcrumbs.any? { |crumb| crumb[:is_page_parent] } ? "collapse-on-mobile" : ""
+  invert_class = inverse ? "govuk-breadcrumbs--inverse" : ""
 %>
 
-<div
-  class="govuk-breadcrumbs <%= "collapse-on-mobile" if collapse_on_mobile && breadcrumbs.any? { |crumb| crumb[:is_page_parent] } %>"
-  data-module="track-click">
+<div class="govuk-breadcrumbs <%= collapse_class %>" data-module="track-click">
   <ol>
   <% breadcrumbs.each_with_index do |crumb, index| %>
     <%
       is_link = crumb[:url].present? || crumb[:is_current_page]
       path = crumb[:is_current_page] ? '#content' : crumb[:url]
       aria_current = crumb[:is_current_page] ? 'page' : 'false'
-      css_class = crumb[:is_current_page] ? 'breadcrumb-for-current-page' : ''
+      css_class = crumb[:is_current_page] ? 'breadcrumb-for-current-page ' : ''
+      css_class << invert_class
```

Comment:
Now that this is always applied, we can probably these 2 lines into one:

```
invert_class.concat(crumb[:is_current_page] ? ' breadcrumb-for-current-page ' : '')
```

Reply:
Done that, too. Thanks!